### ISSUE TITLE
Unable to load version

### ISSUE THREAD

**Issue by baka4n** [DemonWav/MinecraftDevIntelliJ#1939]
### Minecraft Development for IntelliJ plugin version

5.1.22

### IntelliJ version

2022.3.2

### Operating System

windows

### Target platform

Architectury

### Description of the bug

![a](https://user-images.githubusercontent.com/60695447/218497551-269c47d1-f64c-4ad7-9b05-ce2105cda565.png)

**Comment by byBackfish** [comment]
This is the error that pops up when it fails

```Failed to update versions form

java.lang.IllegalArgumentException: Failed to parse version part as integer: 23w03a (whole version text: 23w03a)
	at com.demonwav.mcdev.util.SemanticVersion$Companion.parse$parseInt(SemanticVersion.kt:98)
	at com.demonwav.mcdev.util.SemanticVersion$Companion.parse(SemanticVersion.kt:153)
	at com.demonwav.mcdev.platform.architectury.version.ArchitecturyVersion$Companion.downloadData(ArchitecturyVersion.kt:69)
	at com.demonwav.mcdev.platform.architectury.creator.ArchitecturyProjectSettingsWizard$downloadVersions$2$architecturyApiVersionJob$1.invokeSuspend(ArchitecturyProjectSettingsWizard.kt:258)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.internal.LimitedDispatcher.run(LimitedDispatcher.kt:42)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:95)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
```

**Comment by T3sT3ro** [comment]
Indeed. Happens for `2022.3-1.5.22` with the exact same log, which makes it impossible to create a new project.

It works on `2022.3-1.5.21` plugin version (has to be installed manually). The culprit is most likely [this change](https://github.com/minecraft-dev/MinecraftDev/commit/1adc227fe44e87347ef414d449b45b3d23e8082b?diff=split#r102062507).

As noted by Earthcomputer, will be fixed when new wizard gets merged: #1933
